protocols/relay/src/behaviour/handler.rs

```diff
@@ -549,26 +502,26 @@ enum ReservationRequestFuture {
 
 type Futures<T> = FuturesUnordered<BoxFuture<'static, T>>;
 
+pub(crate) type CHEvent = ConnectionHandlerEvent<
```

Comment:
We don't abbreciate `ConnectionHandler` as `CH` anywhere else in the codebase. How about the following?

```suggestion
pub(crate) type ConnectionHandlerEvent = libp2p_swarm::ConnectionHandlerEvent<
```

Would just require the removal of the `use libp2p_swarm::ConnectionHandler` above.

Reply:
Renamed to 'RelayConnectionHandlerEvent' otherwise it is confusing with `ConnectionHandlerEvent` from `libp2p_swarm` in the code.